SConstruct

```diff
@@ -176,6 +183,44 @@ env = Environment(
   ]
 )
 
+qt_env = None
```

Comment:
@pd0wm any reason this is here instead of `selfdrive/ui/SConscript`?

Reply:
I assume we will have more qt binaries for things like spinner, installer, etc..